test/e2e/e2e_suite.go

```diff
@@ -204,6 +206,42 @@ var _ = Describe("operator-sdk", func() {
 			_, err = tc.Run(runPkgManCmd)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("running basic scorecard tests")
+			var scorecardOutput v1alpha3.TestList
+			runScorecardCmd := exec.Command(tc.BinaryName, "scorecard", "bundle",
+				"--selector=suite=basic",
+				"--output=json",
+				"--skip-cleanup=true",
+				"--wait-time=40s")
+			scorecardOutputBytes, err := tc.Run(runScorecardCmd)
+			Expect(err).NotTo(HaveOccurred())
+			err = json.Unmarshal(scorecardOutputBytes, &scorecardOutput)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(len(scorecardOutput.Items)).To(Equal(1))
+			Expect(scorecardOutput.Items[0].Status.Results[0].State).To(Equal(v1alpha3.PassState))
+
+			By("running olm scorecard tests")
+			runOLMScorecardCmd := exec.Command(tc.BinaryName, "scorecard", "bundle",
+				"--selector=suite=olm",
+				"--output=json",
+				"--skip-cleanup=true",
+				"--wait-time=40s")
+			scorecardOutputBytes, err = tc.Run(runOLMScorecardCmd)
+			Expect(err).To(HaveOccurred())
+			err = json.Unmarshal(scorecardOutputBytes, &scorecardOutput)
+			Expect(err).NotTo(HaveOccurred())
+
+			resultTable := make(map[string]v1alpha3.State)
+			resultTable["olm-status-descriptors"] = v1alpha3.FailState
+			resultTable["olm-crds-have-resources"] = v1alpha3.FailState
+			resultTable["olm-bundle-validation"] = v1alpha3.PassState
+			resultTable["olm-spec-descriptors"] = v1alpha3.FailState
+			resultTable["olm-crds-have-validation"] = v1alpha3.PassState
+
+			for a := 0; a < len(scorecardOutput.Items); a++ {
```

Comment:
Use test names to index into the `resultTable` map to make sure each test is run.